worker.py

```diff
@@ -47,3 +63,22 @@ def notify_task_completion(self, task):
     def sigterm_handler(self, signum, frame):
         logging.info("Exiting gracefully.")
         sys.exit(0)
+
+    def decode_argument(self, arg):
```

Comment:
Can we move the operation methods, decode logic and operations mapping(dict) to a separate class/file (eg. operations.py) and consume it here? 
Note: This is a clean code suggestion. Could be taken up in subsequent PRs.

Reply:
Thanks for the suggestion. Can take up if we have more functions